packages/api/src/hooks/useClientCountry.ts

```diff
@@ -0,0 +1,22 @@
+import { useMemo } from 'react';
+import useQuery from '../useQuery';
+
+/** A custom hook that gets the client country. */
+const useClientCountry = () => {
```

Comment:
Pls make this a generic hook for website_status 😸

Reply:
@thisyahlen-deriv ok, it makes sense, but let's do it as a separate task, in my opinion it's a little bit out of the scope of this card. Also, don't want to rewrite unit tests for this hook ))))

Reply:
hahahah alrights